DESCRIPTION

```diff
@@ -90,8 +90,9 @@ Suggests:
 URL: http://shiny.rstudio.com
 BugReports: https://github.com/rstudio/shiny/issues
 Remotes:
+    tidyverse/ggplot2,
```

Comment:
Does this mean we wouldn't be able to release Shiny to CRAN without a ggplot2 release first? If so, any idea of a timeline @hadley?

Reply:
to work with the `aes()` I believe we'll have to wait.  But that's only for the example.  

If there is a different example, we could use it instead to release sooner.

Reply:
We're in the middle of the release process, with submission scheduled for June 25.

Reply:
(But equally it would be fine to eliminate that example — you can't replace it with anything particularly useful in released ggplot2)